ArduinoCode/Pedal_Box/Pedal_Box.ino

```diff
@@ -97,9 +115,13 @@ void readStates() {
       invertorOn = buf[0]; // set inverter state
     } else if (canId == CAN_DIRECTION) {
       isForward = buf[0]; // set direction state
-    } else if (canId = CAN_BMS_STATES) {
+    } else if (canId == CAN_BMS_STATES) {
       dischargeEnabled = buf[5] & 1;     // get first bit of Relay State byte
       chargeEnabled = (buf[5] >> 1) & 1; // get second bit of Relay State byte 
+    } else if (canId == CAN_MC_FAULTS) {
+      if ((buf[4] != 0) || ((buf[5] >> 3) & 1) || ((buf[5] >> 4) & 1)) { // only send message for fault bits 32-39, 43, 44
```

Comment:
this kind of code formatting of having multiple or statements in an if condition is kinda tough to read. I would recommend splitting it up in these cases so it looks something more like this, but it's up to you:
```
bool conditionA = buf[4] != 0;
bool conditionB = (buf[5] >> 3) & 1;
bool conditionC = (buf[5] >> 4) & 1;
if (conditionA || conditionB || conditionC) {
  ... insert code ...
}
```

Reply:
Not recommended imo. Remember we are working on a lil Arduino. So now you're making 3 more variables that aren't needed and tbh idk enough about the compiler to recall if it actually simplifies those or not.